src/hw9/ArrayDataException.java

```diff
@@ -1,6 +1,6 @@
 package hw9;
 
-public class ArrayDataException extends Throwable{
-    public ArrayDataException ( String s ){
+public class ArrayDataException extends Throwable {
```

Comment:
Extending from `Throwable` is not really correct. Would be enough to extend from `RuntimeException`